tutorials/01-conda_astropy_install.rst

```diff
@@ -0,0 +1,138 @@
+Installing Astropy and Related Packages with Anaconda
+=====================================================
+
+Pre-requisites
+--------------
+
+**Mac** Have the latest version of `Xcode
+developer <https://developer.apple.com/xcode/>`__ tools installed
+
+**Windows** Be able to access a terminal, either through a `Linux
+subsystem <https://docs.microsoft.com/en-us/windows/wsl/install-win10>`__
+or by installing the `Linux bash
+shell <https://www.howtogeek.com/249966/how-to-install-and-use-the-linux-bash-shell-on-windows-10/>`__
+
+Step 1: Download Anaconda
+-------------------------
+
+The Anaconda Python distribution can be downloaded from
+https://www.anaconda.com/download
+
+-  The scientific computing community is now using Python 3 as a
+   default.
+
+**But my code only runs in Python 2** Please see the next tutorial: *An
+Astropy User's Guide to Managing Conda Environments*
+
+-  When the download is finished, click on the package and follow the
+   installation instructions.
+
+-  Open a terminal window to check that the Anaconda installation will
+   work for you:
+
+   ::
+
+       which conda
+
+   should return something like ``/anaconda3/bin/conda`` and
+
+   ::
+
+       which python
+
+   should return a Python path that is in the same directory as
+   Anaconda: ``/anaconda3/bin/python``
+
+Step 2: Install core packages
+-----------------------------
+
+The default Anaconda installation comes with many packages that
+astronomers use frequently: *numpy*, *scipy*, and *matplotlib*
+
+We can use the ``conda install`` command to install everything else we
+need. Anaconda will automatically check, update, and install any python
+packages that your desired package depends on.
+
+-  Below, we give an example of installing astropy along with some common 
+   scientific, statistical, and visualization packages. You can install them all
+   individually or in one line:
+
+::
+
+    conda install astropy matplotlib scikit-learn pandas
+
+Step 3: Install affiliated packages
+-----------------------------------
+
+Many `Astropy affiliated
+packages <https://www.astropy.org/affiliated/>`__ can be found on
+*astroconda* channel, maintained by AURA and STScI. To add this channel
```

Comment:
I would prefer the `astropy` channel (which is what the command below adds) for two reasons:

+ The `astropy` channel goal is to provide all of the affiliated packages.
+ The `astropy` channel includes builds for Windows whenever possible.